pkg/plugins/golang/v3/init.go

```diff
@@ -121,10 +125,10 @@ func (p *initSubcommand) PreScaffold(machinery.Filesystem) error {
 }
 
 func (p *initSubcommand) Scaffold(fs machinery.Filesystem) error {
-	scaffolder := scaffolds.NewInitScaffolder(p.config, p.license, p.owner)
+	scaffolder := scaffolds.NewWorkspaceInitScaffolder(
```

Comment:
I do not think that we should change the method to initScaffolder, any required config ought to be in the PROJECT file.